aspnetcore/signalr/messagepackhubprotocol.md

```diff
@@ -104,6 +104,26 @@ const connection = new signalR.HubConnectionBuilder()
 > [!NOTE]
 > At this time, there are no configuration options for the MessagePack protocol on the JavaScript client.
 
+### Java client
+
+To enable MessagePack in the Java, first install the `com.microsoft.signalr.messagepack` package. If using Gradle, add the following line to the `dependencies` section of your *build.gradle* file:
+
+```gradle
+implementation 'com.microsoft.signalr.messagepack:signalr-messagepack:5.0.0'
+```
+
+If using Maven, add the following lines inside the `<dependencies>` element of your *pom.xml* file:
+
+[!code-xml[pom.xml dependency element messagePack](java-client/sample/pom.xml?name=snippet_dependencyElement_messagePack)]
```

Comment:
This didn't work, anybody know what the right syntax is?

Reply:
Looks like it worked to me: https://review.docs.microsoft.com/en-us/aspnet/core/signalr/messagepackhubprotocol?view=aspnetcore-5.0&branch=pr-en-us-21090#java-client